README.md

```diff
@@ -79,7 +79,7 @@ In order to run this project you need:
 ### Setup
 
 Clone this repository to your desired folder:
-- Clone [this repository](https://github.com/Kwaleyela-Ikafa/blog-app.git)
+- Clone [this repository](https://github.com/Kwaleyela-Ikafa/budget-app.git)
```

Comment:
- [ ] Please give credit to the author of the original design in the `Acknowledgement` section as it is in the project section